prov/verbs_ud/src/verbs_ud.h

```diff
@@ -0,0 +1,738 @@
+/*
+ * Copyright (c) 2017 Intel Corporation, Inc.  All rights reserved.
+ *
+ * This software is available to you under a choice of one of two
+ * licenses.  You may choose to be licensed under the terms of the GNU
+ * General Public License (GPL) Version 2, available from the file
+ * COPYING in the main directory of this source tree, or the
+ * BSD license below:
+ *
+ *     Redistribution and use in source and binary forms, with or
+ *     without modification, are permitted provided that the following
+ *     conditions are met:
+ *
+ *      - Redistributions of source code must retain the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer.
+ *
+ *      - Redistributions in binary form must reproduce the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer in the documentation and/or other materials
+ *        provided with the distribution.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+ * BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#ifndef _OFI_VERBS_UD_H_
+#define _OFI_VERBS_UD_H_
+
+#if HAVE_CONFIG_H
+#  include <config.h>
+#endif /* HAVE_CONFIG_H */
+
+#include <infiniband/verbs.h>
+#include <rdma/rdma_cma.h>
+
+#include <fi_util.h>
+
+#include <inttypes.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+#define OFI_VERBS_UD_MAJOR_VERSION 1
+#define OFI_VERBS_UD_MINOR_VERSION 5
+
+#define OFI_VERBS_UD_PRIMARY_CAPS (FI_MSG | FI_RMA |	\
+				   FI_SEND | FI_RECV)
+
+#define OFI_VERBS_UD_CAPS OFI_VERBS_UD_PRIMARY_CAPS
+
+#define VERBS_UD_MSG_IOV_LIMIT	256
```

Comment:
Default. Ok, I agree with you.
Please, review `prov/verbs/fi_verbs.c`. The existed variable of prov/verbs `def_iov_limit` is used from now.